xtevent/_eventols.ado

```diff
@@ -98,23 +98,22 @@ program define _eventols, rclass
 	
 	* error messages for sun_abraham
 	loc sun_abraham ""
-	if "`cohort'"!="" & "`control_cohort'"!="" {
-		di as text _n "You have specified {bf:cohort} and {bf:control_cohort} options."
-		di as text _n "Event-time coefficients will be estimated with"
-		di as text _n "the Interaction Weighted Estimator of Sun and Abraham (2021)."
+	if "`cohort'"!="" {
+		di as text _n "You have specified the {bf:cohort} or the {bf:sunabraham} option"
+		di as text "Event-time coefficients will be estimated with the Interaction Weighted Estimator of Sun and Abraham (2021)"
 		loc sun_abraham "sun_abraham"
 	}
 	if "`saveint'"!="" & "`sun_abraham'"==""{
-		di as err _n "Suboption {bf:saveint} can only be specified if options {bf:cohort} and {bf:control_cohort} have been specified as well."
+		di as err _n "Suboption {bf:saveint} can only be specified with {bf:cohort}"
 		exit 301
 	}
 	* Check for vars named _interact

```

Comment:
@jorpppp 
When I run the following:
```
use example31, clear
xtevent y eta , panelvar(i) timevar(t) policyvar(z) window(10) impute(stag) savek(aa, saveinteract) sunabraham
```
I get the following [error message](https://github.com/JMSLab/xtevent/assets/36095845/344dded8-ab76-40d7-8f32-3eec06e84072).

Reply:
@Constantino-Carreto-Romero got it, let me check.

Reply:
Fixed in https://github.com/JMSLab/xtevent/pull/185/commits/aeb06643c92216b813b3369eb67984d73e19f880